app/views/hosts/_list.html.erb

```diff
@@ -15,10 +18,24 @@
   </thead>
   <tbody>
     <% hosts.each do |host| %>
-      <tr>
+      <tr data-host-id=<%= host.id %>>
         <td class="ca">
           <%= check_box_tag "host_ids[]", nil, false, :id => "host_ids_#{host.id}", :class => 'host_select_boxes', :onclick => 'hostChecked(this)' %>
         </td>
+        <% if power_status_visible? %>
+          <td class="ca">
+            <%= spinner '', { :class => "fn" } %>
+            <span class="label label-default hidden">
```

Comment:
also, maybe we should just return the html from the server vs have all options here in the first place with hide/show logic?